File: SpringCloudDev/springcloud-vue-ui/src/entitys/integral/marketingact.js
                                                                                    export default function area() {
                                                                                      return {
                                                                                        current: 1,
                                                                                        size: 20,
                                                                                        isNewRecord: false,
                                                                                        query: {
                                                                                          // 表id
                                                                                          marketingActId: undefined,
                                                                                          // 活动CODE
                                                                                          actCode: undefined,
                                                                                          // 活动名称
                                                                                          actName: undefined,
                                                                                          // 活动开始时间
                                                                                          beginTime: undefined,
                                                                                          // 活动结束时间
                                                                                          endTime: undefined,
                                                                                          // 卡券编号
                                                                                          cardTieketId: undefined,
                                                                                          // 积分类型
                                                                                          integralType: undefined,
                                                                                          // 积分类型ID
                                                                                          integralId: undefined,
                                                                                          // 活动链接
                                                                                          actLink: undefined,
                                                                                          // 活动状态
                                                                                          actStatus: undefined,
                                                                                          // 活动状态ID
                                                                                          actStatusId: undefined,
                                                                                          // 活动说明
                                                                                          actExplain: undefined,
                                                                                          // 活动目标描述
                                                                                          actTargetBreif: undefined,
                                                                                          // 活动卡券数量
                                                                                          actCardTicketNum: undefined,
                                                                                          // 活动目标类型
                                                                                          actTargetType: undefined,
                                                                                          // 活动目标数量
                                                                                          actTargetNumNum: undefined,
                                                                                          // 活动积分数量
                                                                                          actIntegralNumNum: undefined,
                                                                                          // 审核评语
                                                                                          reviewComments: undefined,
                                                                                          // 预留字段一
                                                                                          reserveColumn1: undefined,
                                                                                          // 预留字段二
                                                                                          reserveColumn2: undefined,
                                                                                          // 预留字段三
                                                                                          reserveColumn3: undefined
                                                                                        },
                                                                                        rules: {
                                                                                          marketingActId: [{
                                                                                            required: true,
                                                                                            message: '请输入表id',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actCode: [{
                                                                                            required: true,
                                                                                            message: '请输入活动CODE',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actName: [{
                                                                                            required: true,
                                                                                            message: '请输入活动名称',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          beginTime: [{
                                                                                            required: true,
                                                                                            message: '请输入活动开始时间',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          endTime: [{
                                                                                            required: true,
                                                                                            message: '请输入活动结束时间',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          cardTieketId: [{
                                                                                            required: true,
                                                                                            message: '请输入卡券编号',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          integralType: [{
                                                                                            required: true,
                                                                                            message: '请输入积分类型',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          integralId: [{
                                                                                            required: true,
                                                                                            message: '请输入积分类型ID',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actLink: [{
                                                                                            required: true,
                                                                                            message: '请输入活动链接',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actStatus: [{
                                                                                            required: true,
                                                                                            message: '请输入活动状态',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actStatusId: [{
                                                                                            required: true,
                                                                                            message: '请输入活动状态ID',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actExplain: [{
                                                                                            required: true,
                                                                                            message: '请输入活动说明',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actTargetBreif: [{
                                                                                            required: true,
                                                                                            message: '请输入活动目标描述',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actCardTicketNum: [{
                                                                                            required: true,
                                                                                            message: '请输入活动卡券数量',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actTargetType: [{
                                                                                            required: true,
                                                                                            message: '请输入活动目标类型',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actTargetNumNum: [{
                                                                                            required: true,
                                                                                            message: '请输入活动目标数量',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          actIntegralNumNum: [{
                                                                                            required: true,
                                                                                            message: '请输入活动积分数量',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          reviewComments: [{
                                                                                            required: true,
                                                                                            message: '请输入审核评语',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          reserveColumn1: [{
                                                                                            required: true,
                                                                                            message: '请输入预留字段一',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          reserveColumn2: [{
                                                                                            required: true,
                                                                                            message: '请输入预留字段二',
                                                                                            trigger: 'blur'
                                                                                          }],
                                                                                          reserveColumn3: [{
                                                                                            required: true,
                                                                                            message: '请输入预留字段三',
                                                                                            trigger: 'blur'
                                                                                          }]
                                                                                        }
                                                                                      }
                                                                                    }

File: SpringCloudDev/springcloud-business-api/springcloud-integral-api/src/main/java/org/springclouddev/integral/entity/MarketingAct.java
package org.springclouddev.integral.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.experimental.Accessors;
import org.springclouddev.core.mp.base.BaseEntity;

/**
 * 实体类
 *
 * @author zhaobohao
 * @since 2020-01-15
 */
@Data
@Accessors(chain = true)
@TableName("tbl_marketing_act")
@EqualsAndHashCode(callSuper = true)
@ApiModel(value = "MarketingAct对象" , description = "MarketingAct对象")
public class MarketingAct extends BaseEntity {

    private static final long serialVersionUID = 1L;

    /**
     * 表id
     */
    @ApiModelProperty(value = "表id")
    private Long id;
    /**
     * 活动CODE
     */
    @ApiModelProperty(value = "活动CODE")
    @TableField("ACT_CODE")
    private String actCode;
    /**
     * 活动名称
     */
    @ApiModelProperty(value = "活动名称")
    @TableField("ACT_NAME")
    private String actName;
    /**
     * 活动开始时间
     */
    @ApiModelProperty(value = "活动开始时间")
    @TableField("BEGIN_TIME")
    private String beginTime;
    /**
     * 活动结束时间
     */
    @ApiModelProperty(value = "活动结束时间")
    @TableField("END_TIME")
    private String endTime;
    /**
     * 卡券编号
     */
    @ApiModelProperty(value = "卡券编号")
    @TableField("CARD_TIEKET_ID")
    private String cardTieketId;
    /**
     * 积分类型
     */
    @ApiModelProperty(value = "积分类型")
    @TableField("INTEGRAL_TYPE")
    private String integralType;
    /**
     * 积分类型ID
     */
    @ApiModelProperty(value = "积分类型ID")
    @TableField("INTEGRAL_ID")
    private String integralId;
    /**
     * 活动链接
     */
    @ApiModelProperty(value = "活动链接")
    @TableField("ACT_LINK")
    private String actLink;
    /**
     * 活动状态
     */
    @ApiModelProperty(value = "活动状态")
    @TableField("ACT_STATUS")
    private String actStatus;
    /**
     * 活动状态ID
     */
    @ApiModelProperty(value = "活动状态ID")
    @TableField("ACT_STATUS_ID")
    private String actStatusId;
    /**
     * 活动说明
     */
    @ApiModelProperty(value = "活动说明")
    @TableField("ACT_EXPLAIN")
    private String actExplain;
    /**
     * 活动目标描述
     */
    @ApiModelProperty(value = "活动目标描述")
    @TableField("ACT_TARGET_BREIF")
    private String actTargetBreif;
    /**
     * 活动卡券数量
     */
    @ApiModelProperty(value = "活动卡券数量")
    @TableField("ACT_CARD_TICKET_NUM")
    private Long actCardTicketNum;
    /**
     * 活动目标类型
     */
    @ApiModelProperty(value = "活动目标类型")
    @TableField("ACT_TARGET_TYPE")
    private String actTargetType;
    /**
     * 活动目标数量
     */
    @ApiModelProperty(value = "活动目标数量")
    @TableField("ACT_TARGET_NUM_NUM")
    private Long actTargetNumNum;
    /**
     * 活动积分数量
     */
    @ApiModelProperty(value = "活动积分数量")
    @TableField("ACT_INTEGRAL_NUM_NUM")
    private Long actIntegralNumNum;
    /**
     * 审核评语
     */
    @ApiModelProperty(value = "审核评语")
    @TableField("REVIEW_COMMENTS")
    private String reviewComments;
    /**
     * 预留字段一
     */
    @ApiModelProperty(value = "预留字段一")
    @TableField("RESERVE_COLUMN1")
    private String reserveColumn1;
    /**
     * 预留字段二
     */
    @ApiModelProperty(value = "预留字段二")
    @TableField("RESERVE_COLUMN2")
    private String reserveColumn2;
    /**
     * 预留字段三
     */
    @ApiModelProperty(value = "预留字段三")
    @TableField("RESERVE_COLUMN3")
    private String reserveColumn3;


}
